Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8
+FROM python:3.8-slim
```

Comment:
The image does not include curl/wget which we might need to install node, postgres-client later. Not sure if it's a better practice to take a larger image or install more stuff by hand. Might be adapted again in multi stage build anyways.

Reply:
Front build will be done in another image, let's see later for the postgres-client.